apps/embed/src/components/comments/queries/submitPostComment.ts

```diff
@@ -224,9 +226,37 @@ async function postCommentWithoutGasless({
   }
 
   try {
+    let fee:
+      | Awaited<ReturnType<typeof estimateChannelPostCommentFee>>
+      | undefined;
+
+    if (channelId) {
+      // Estimate the fee for posting a comment to the channel
+      const estimationCommentData =
+        createEstimateChannelPostOrEditCommentFeeData({
+          ...postCommentPayloadRequest,
+          app: publicEnv.NEXT_PUBLIC_APP_SIGNER_ADDRESS,
+        });
+      fee = await estimateChannelPostCommentFee({
+        commentData: estimationCommentData,
+        metadata: [],
+        msgSender: author,
+        readContract: readContractAsync,
+        channelId,
+      });
+    }
+
+    if ((fee?.contractAsset?.amount ?? 0n) > 0n) {
+      // TODO: this will be supported in a separate PR or ticket
+      throw new SubmitPostCommentMutationError(
+        "We don't support posting to a channel requires fees to be paid in contract assets yet.",
```

Comment:
```suggestion
        "We don't support posting to a channel that requires fees to be paid in contract assets yet.",
```

is this what you meant?